py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -338,6 +338,12 @@ def format_data(self,
                     "z": row['Z'],
                 }
                 args["log_lambda"] = np.log10(spec['WAVELENGTH'])
+                if "TILEID" in row:
+                    args["tile"] = row['TILE']
+                if "PETAL_LOC" in row:
+                    args["petal"] = row['PETAL_LOC']
+                if "NIGHT" in row:
+                    args["night"] = row['NIGHT']
 
```

Comment:
Should we add e.g. FIBER or EXPID? Are these fields all read earlier from HDU2 (EXP_FIBERMAP) of the coadd-files, or just when reading spectra-files?

Reply:
Just checked and here we'd just copy over info from the catalogue, is all this info in the actual quasar cats? or would we need to assemble the data from the coadd-files?